docs/application/tizen-studio/rt-ide/get-started/create.md

```diff
@@ -1,6 +1,7 @@
 # Managing a Project
 
-You can create an RT project in Tizen Studio. Additionally, you can create application resources within the project and build the project.
+> [!NOTE] 
+> Tizen RT-IDE has been deprecated since Tizen Studo release 4.1. It is not recommended to use RT-IDE. No support will be provided for RT-IDE after Nov 23, 2020.

```

Comment:
This date is confirmed with IDE team

Reply:
@annie-samsung I took the dates from release notes, and the date was confirmed by @Manish previously.

Reply:
@manishtr could you confirm?

Reply:
the content is fine , the release date of 4.1 is not fixed, we can modify the dates accordingly

Reply:
@manishtr , sure thanks

Reply:
How about using more concise expression?

```suggestion
> Tizen RT-IDE has been deprecated since Tizen Studio 4.1, so no support will be provided.
```

or, simple below form can be used as like [other page case](https://docs.tizen.org/application/tizen-studio/native-tools/ui-builder/ui-builder/).

```suggestion
> Tizen RT-IDE has been deprecated since Tizen Studio 4.1.
```

Until the release date, updating every pages every time when the release date is changed is a burden. I think reducing it would be better.


Also, are you going to add the note to every pages, not only to the first page?

Reply:
@mijin85cho I have added the note to every page.